docs/validators/upgrades/v5.0-upgrade.md

```diff
@@ -0,0 +1,94 @@
+# Upgrade Guide v5.0
+
+This document provides guidelines for node operators upgrading from Regen Ledger v5.0 to v5.0. For a general overview of the upgrade process, please refer to [Upgrade Overview](README.md).
+
+## Upgrade Info
+
+### Regen Ledger v5.0
+
+- [Release Notes](https://github.com/regen-network/regen-ledger/releases/tag/v5.0.0)
+
+### Regen Mainnet
+
+- Upgrade Height: `TBD`
+
+### Redwood Testnet
+
+- Upgrade Height: `TBD`
+
+## Upgrade Guidelines
+
+### Update Go
+
+**Go `1.18` or later is required.**
+
+For available downloads, see [Go Downloads](https://go.dev/dl/).
+
+### Option 1: Using Cosmovisor
+
+**Cosmovisor `v1.0` or later is required.**
+
+The following instructions assume the `cosmovisor` binary is already installed and `cosmovisor` is set up as a systemd service. If this is not the case, please refer to [Using Cosmovisor](../get-started/using-cosmovisor.md) for instructions on how to install and set up `cosmovisor`.
+
+::: warning Auto-Download Not Recommended
+Cosmovisor provides an auto-download option that prevents the need for the following steps in preparing for the upgrade. Although available to validators, this feature should only be used for non-validator nodes and validators should build the binary from source.
+:::
+
+Build the upgrade binary (`v5.0.0`) from source:
+
+```bash
+cd regen-ledger
+git fetch --all
+git checkout v5.0.0
+make build
+```
+
+Ensure the `regen` binary has been built:
+
+```bash
+./build/regen version
+```
+
+You should see the following:
+
+```bash
+v5.0.0
+```
+
+Create a `v5.0.0` directory and copy the upgrade binary (`v5.0.0`) to the directory:
+
+```bash
+mkdir -p $HOME/.regen/cosmovisor/upgrades/v5.0.0/bin
+cp ./build/regen $HOME/.regen/cosmovisor/upgrades/v5.0.0/bin
+```
+
+At the proposed block height, `cosmovisor` will automatically stop the current binary (`v4.0.0`), set the upgrade binary as the current binary (`v4.0.0`), and then (depending on the cosmovisor settings) perform a backup and restart the node.
```

Comment:
```suggestion
At the proposed block height, `cosmovisor` will automatically stop the current binary (`v4.x`), set the upgrade binary as the current binary (`v4.0.0`), and then (depending on the cosmovisor settings) perform a backup and restart the node.
```
Let's call it `v4.x`? We are currently on v4.1.4, but just incase we update with another minor/patch release before now and when we do v5.0.